### DISCUSSION TITLE
How to avoid browser blocking downloads from my site?

### DISCUSSION THREAD

**Discussion by fadethegap** [supabase/supabase#6503] [Questions]
Our application enables subscribers to download video editing assets. Some of our assets are similar and we package them with in a zip file. Starting with Chrome 84, Google started blocking files from websites not using https://. 

[https://www.pixelstech.net/article/1600480115-Chrome-to-block-mixed-content-downloads-in-version-86](https://www.pixelstech.net/article/1600480115-Chrome-to-block-mixed-content-downloads-in-version-86)

Here is a sample link: https://foxbbdebuzofwrfvzazi.supabase.co/storage/v1/object/public/assets/AssetPack1.zip. 

Any ideas what I need to do to allow the download? It is clearly coming from https.

**Comment by GaryAustin1** [comment]
Are you saying clicking on your link does not download?  Downloads fine in Chrome/Firefox/Edge for me.

**Reply by fadethegap** [reply]
Dang it! Crummy example. I am refactoring. I will return with an 'actual' example.

Thanks @GaryAustin1

**Reply by fadethegap** [reply]
@GaryAustin1 I believe I figured it out. I was trying to download from my localhost which is http, and google now prevents downloading https items from an http source.
